recipes/oauthenticator/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "oauthenticator" %}
```

Comment:
There is a naming convention discrepancy here that starts on the jupyterhub side. `oauthenticator` is the proper name of the package in PyPi, but for the `ldapauthenticator`, the actual PyPi package is called `jupyterhub-ldapauthenticator`.

@conda-forge/help-python, what should we do in this case?

Reply:
I think using the same name as on PyPI is probably less confusing to people.